src/App.js

```diff
@@ -20,8 +20,9 @@ function App() {
           <Route path="/profile" element={<Profile />} />
           <Route path="/memberList" element={<MemberList />} />
           <Route path="/managerList" element={<ManagerList />} />
+          <Route path="/managerManagment" element={<ManagerManagment />} />
           <Route path="/eventList" element={<EventList />} />
-          <Route path="/event" element={<Event />} />
+          <Route path="/event" element={<ManagerList />} />
```

Comment:
<Route path="/event" element={<ManagerList />} />
  
  maybe we should change path="/event" to path="/managerList" ?